keps/prod-readiness/sig-apps/4368.yaml

```diff
@@ -0,0 +1,5 @@
+kep-number: 4368
+alpha:
```

Comment:
You probably should drop alpha as you are going to beta directly.

Reply:
right, done